queries.py

```diff
@@ -480,6 +506,147 @@ def construct_select_original_bericht_query(bericht_uri):
+    PREFIX ere:         <http://data.lblod.info/vocabularies/erediensten/>
+    PREFIX org:         <http://www.w3.org/ns/org#>
+    PREFIX pav:         <http://purl.org/pav/>
+    PREFIX meb:         <http://rdf.myexperiment.org/ontologies/base/>
+    PREFIX dct:         <http://purl.org/dc/terms/>
+    PREFIX prov:        <http://www.w3.org/ns/prov#>
+    PREFIX adms:        <http://www.w3.org/ns/adms#>
+    
+    ASK {{
+        BIND(<{0}> AS ?submission)
+        ?bestuurseenheid a ere:RepresentatiefOrgaan .
+        ?submission a meb:Submission ;
+                   adms:status <http://lblod.data.gift/concepts/9bd8d86d-bb10-4456-a84e-91e9507c374c> ;
+                   prov:generated ?formData ;
+                   pav:createdBy ?bestuurseenheid .
+        ?formData dct:type ?decisionType .
+        VALUES ?decisionType {{ {1} }}
+    }}
+    """.format(submission, " ".join(DECISION_TYPES_RO))
+
+    return ask_query_ro
+
+def verify_go_exclusion_rule(submission)
+
+    submission = escape_helpers.sparql_escape_string(submission)
+
+    ask_query_go = """
+    PREFIX ere:         <http://data.lblod.info/vocabularies/erediensten/>
+    PREFIX org:         <http://www.w3.org/ns/org#>
+    PREFIX pav:         <http://purl.org/pav/>
+    PREFIX meb:         <http://rdf.myexperiment.org/ontologies/base/>
+    PREFIX dct:         <http://purl.org/dc/terms/>
+    PREFIX besluit:     <http://data.vlaanderen.be/ns/besluit#>
+    PREFIX prov:        <http://www.w3.org/ns/prov#>
+    PREFIX adms:        <http://www.w3.org/ns/adms#>
+    
+    ASK {{
+        BIND(<{0}> AS ?submission)
+        ?bestuurseenheid besluit:classificatie <http://data.vlaanderen.be/id/concept/BestuurseenheidClassificatieCode/5ab0e9b8a3b2ca7c5e000001> .
+        ?submission a meb:Submission ;
+                   adms:status <http://lblod.data.gift/concepts/9bd8d86d-bb10-4456-a84e-91e9507c374c> ;
+                   prov:generated ?formData ;
+                   pav:createdBy ?bestuurseenheid .
+        ?formData dct:type ?decisionType .
+        VALUES ?decisionType {{ {1} }}
+    }}
+    """.format(submission, " ".join(DECISION_TYPES_GO))
+
+    return ask_query_go
+
+def verify_po_exclusion_rule(submission)
+
+    submission = escape_helpers.sparql_escape_string(submission)
+
+    ask_query_po = """
+    PREFIX ere:         <http://data.lblod.info/vocabularies/erediensten/>
+    PREFIX org:         <http://www.w3.org/ns/org#>
+    PREFIX pav:         <http://purl.org/pav/>
+    PREFIX meb:         <http://rdf.myexperiment.org/ontologies/base/>
+    PREFIX dct:         <http://purl.org/dc/terms/>
+    PREFIX besluit:     <http://data.vlaanderen.be/ns/besluit#>
+    PREFIX prov:        <http://www.w3.org/ns/prov#>
+    PREFIX adms:        <http://www.w3.org/ns/adms#>
+    
+    ASK {{
+        BIND(<{0}> AS ?submission)
+        ?bestuurseenheid besluit:classificatie <http://data.vlaanderen.be/id/concept/BestuurseenheidClassificatieCode/5ab0e9b8a3b2ca7c5e000000> .
```

Comment:
To be sure, can you move this line below the line about the submission. I think there is something about the order of reading the statements that might improve performance. (I am not sure but it does not hurt)